### ISSUE TITLE
Looking for maintainers.

### ISSUE THREAD

**Issue by Tjoosten** [Activisme-be/Blade-github-octicons#37]
Hey everyone,

It's been a while since I contributed to this package. And after a short thought process I decided to look for a co-maintainer for this package.

Due to changing priorities and other matters in my private life, I don't always have the time. To continue to maintain this package.

That's why I decided to step aside in the project.

If you are interested in the maintenance and or later take over the project. Feel free to respond. Or to contact me on [Twitter](https://twitter.com/X0rif).

Follow up on #36

**Comment by swapnilsarwe** [comment]
> Hey everyone,
> 
> It's been a while since I contributed to this package. And after a short thought process I decided to look for a co-maintainer for this package.
> 
> Due to changing priorities and other matters in my private life, I don't always have the time. To continue to maintain this package.
> 
> That's why I decided to step aside in the project.
> 
> If you are interested in the maintenance and or later take over the project. Feel free to respond. Or to contact me on [Twitter](https://twitter.com/X0rif).
> 
> Follow up on #36 

I have a streamlined process for maintaining all my blade-icons packages. I guess I can add those actions in this repo. Which will take all your load keeping your packages updated with the new set from the original icon owner.

Apart from that all dependency upgrades and other stuff, i can keep it up to date through PRs or as a co-maintainer.

Your private life is most important, give it a topmost priority, we are here with you to take care of the wonderful package you are maintaining.

Cheers.

**Comment by Tjoosten** [comment]
Hey @swapnilsarwe, 

I've read your reply. and it sounds good to me if you want to commit is it alright that i make u co-maintainer of this package? 

Kind regards, 
Tim

**Comment by swapnilsarwe** [comment]
> Hey @swapnilsarwe, 
> 
> I've read your reply. and it sounds good to me if you want to commit is it alright that i make u co-maintainer of this package? 
> 
> Kind regards, 
> Tim 

Yes Tim, I will be obliged to take care of this repo along with you.

Yes, I commit.

**Comment by Tjoosten** [comment]
Closed becasue a maintainer has been found. Thanks @swapnilsarwe!
